packages/opentelemetry-core/test/platform/AWSXrayIdGenerator.test.ts

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Really sorry about the confusion, since I have no idea where to put vendor code, I temporarily put code and corresponding tests under the same repo as original id.ts and drafted a PR to get suggestions here.